### PR TITLE
Add more dark mode sites

### PR DESCRIPTION
**Pull request by quick007**

### CODE REVIEW

File: src/config/dark-sites.config [hunk 1]
```diff
@@ -189,6 +189,7 @@ dark.diatr.us
 darkmodelist.com
 darkreader.org
 dash.cavebot.xyz
+dashboard.minefly.org
```
Comment (Gusted) [inline src/config/dark-sites.config:192]: Not a site? I get a error connection.

File: src/config/dark-sites.config [hunk 2]
```diff
@@ -857,6 +859,8 @@ valvestore.forfansbyfans.com
 vancedapp.com
 vanillatweaks.net
 vcjhwebdev.github.io/useless-translator/
+vercel.app
```
Comment (Gusted) [inline src/config/dark-sites.config:862]: Not dark-by-default.

File: src/config/dark-sites.config [hunk 3]
```diff
@@ -857,6 +859,8 @@ valvestore.forfansbyfans.com
...
+vercel.com
```
Comment (Gusted) [inline src/config/dark-sites.config:863]: Not dark-by-default.

File: src/config/dark-sites.config (hunk 1 above)
Comment (quick007) [reply src/config/dark-sites.config:192]: Thanks for catching that, fixed

**Review by Gusted — APPROVED** [review]

File: src/config/dark-sites.config [hunk 4]
```diff
@@ -207,6 +207,7 @@ devanbuggay.com
 devart.withgoogle.com
 developerinsider.co
 developpement-systeme-exploitation.github.io/documentation
+dev.minefly.org
```
Comment (Gusted) [inline src/config/dark-sites.config:210]: This also doesn't seem to add up?
image

**Review by Gusted — CHANGES_REQUESTED** [review]
See comment.

**Comment by quick007** [comment]
Works on my machine, even in incognito 
image

**Comment by rakleed** [comment]
> Works on my machine, even in incognito image

@quick007 you should enable light theme in your operating system. Then the site will also be displayed in a light theme.

**Pull request closed without merging**